modules/dnn/src/layers/fast_convolution/depthwise_convolution.cpp

```diff
@@ -30,6 +30,11 @@ static void depthWiseBlockConv2D(const float* wptr,
             w10 = wptr[3], w11 = wptr[4], w12 = wptr[5],
             w20_ = wptr[6], w21_ = wptr[7], w22_ = wptr[8];
     int outW1 = min(outW, (width - dilation_w*(kernel_w - 1) + pad_l)/stride_w);
+
+    const int VEC_NLANES = 4;
+    if (fusedAdd)
+        outW1 = min(outW, outW1 - outW1%VEC_NLANES);
```

Comment:
Next questions (to show why this doesn't look like a valid change):
- why do we have that outsize of `CV_SIMD`?
- why `VEC_NLANES = 4`. what if we optimize for SIMD256?

Original question was why do we change range of scalar for loops in the SIMD code (which could be enabled or disabled).

Reply:
I got your point. The code has been updated.